validator/nodejs/index.js

```diff
@@ -438,6 +549,7 @@ function main() {
       .then(function(validator) {
         Promise.all(inputs)
             .then(function(resolvedInputs) {
+              resolvedInputs = flatten(resolvedInputs);
```

Comment:
Can you please explain why flatten is needed here? It may be useful to document the type of resolvedInputs, e.g. with a comment. Also perhaps it's a good choice to avoid the dependency on this single function from a third party module.